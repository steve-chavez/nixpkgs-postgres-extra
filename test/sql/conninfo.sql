create extension if not exists conninfo;

select * from conninfo_parse('postgresql://user:pass@localhost:5432/postgres?target_session_attrs=read-write');
