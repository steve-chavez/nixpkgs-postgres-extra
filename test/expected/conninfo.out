create extension if not exists conninfo;
select * from conninfo_parse('postgresql://user:pass@localhost:5432/postgres?target_session_attrs=read-write');
       keyword        |   value    
----------------------+------------
 user                 | user
 password             | pass
 dbname               | postgres
 host                 | localhost
 port                 | 5432
 target_session_attrs | read-write
(6 rows)

